README.md

```diff
@@ -0,0 +1,237 @@
+<p>
+
+
+## Getting Started
+
+The SDK supports iOS 11+.
+
+
+### Adding the SDK dependency
+
+#### Using CocoaPods
+
+The SDK is available on CocoaPods and you can include it in your project by adding the following to your Podfile:
+
+```ruby
+pod 'OnfidoAuth'
+```
+
+Run `pod install` to install the SDK.
+
+#### Using Swift Package Manager
+
+The SDK is available with Swift Package Manager and you can include it in your project by adding the following package repository URL:
+
+https://github.com/onfido/onfido-auth-sdk-ios.git
+
+### App permissions
+
+The SDK makes use of the device’s camera. You will be required to have the `NSCameraUsageDescription` key in your application’s Info.plist file.
+
+```xml
+<key>NSCameraUsageDescription</key>
+<string>Required for facial capture</string>
+```
+
+### Creating the SDK configuration
+Once you have added the SDK as a dependency and you have an applicant ID, you can configure the SDK:
+
+```swift
+let config = try! OnfidoAuthConfig.builder()
+    .withSDKToken("YOUR_SDK_TOKEN_HERE")
+    .withRetryCount(2) // Optional, default is 0
+    .build()
+
+let flow = OnfidoAuthFlow(withConfiguration: config)
+    .with(responseHandler: { response in
+        // Callback when flow ends
+    })
+```
+
+### Starting the flow
+Run the flow from the view controller you want the flow to be presented from e.g. `self`, where `self` is a view controller.
+
+```swift
+flow.run(from: self)
+```
+
+Congratulations! You have successfully started the flow. Read the next sections to learn how to handle callbacks and customise the SDK.
+
+
+## Handling callbacks
+
+To receive the response from the flow, you should pass a callback to the instance of `OnfidoAuthFlow`. Typically, on success, you would create a check on your backend server.
+
+The response object passed to the callback may include the following attributes: `.success(AuthenticationResult)`, `.cancel(CancellationReason)` and `.error(Error)`.
+
+```swift
+let responseHandler: (OnfidoAuthResponse) -> Void = { response in
+    switch response {
+        case .success(let result):
+            // User completed flow
+            // You can create your check here
+        case .cancel(let reason):
+            // Flow cancelled by user
+            // Reason can be:
+            // .deniedConsent -> denied consent on consent screen
+            // .openedInLandscape -> face scan landscape
+            // .openedInReversePortrait -> face scan upside down
+            // .userExited -> tapped back button on first screen
+        case .error(let error):
+            // Error occurred
+    }
+}
+```
+
+### Success handling
+
+Success is when the user has reached the end of the flow. `AuthenticationResult` has the `verified`, `uuid` and `token` properties. If authentication was successful, `verified` will be `true` and `token` will be the JWT. There should always be a `uuid`, whether authentication succeeded or failed.
```

Comment:
Maybe should mention something about being able to verify the JWT with the API endpoint

Reply:
I don't know how the API works. Do you know if we have some information documented somewhere I can look at?

Reply:
https://documentation.onfido.com/#verify-authentication-tokens 

Maybe something like '`token` will be a JWT token which can be used to validate the response'